odnoklassniki-android-sdk/src/main/java/ru/ok/android/sdk/Odnoklassniki.java

```diff
@@ -213,8 +255,8 @@ public void run() {
      * @return query result
      * @throws IOException in case of a problem or the connection was aborted.
      */
-    public final String request(final String apiMethod, final String httpMethod) throws IOException {
-        return request(apiMethod, null, httpMethod);
+    public final String requestSync(final String apiMethod, final String httpMethod) throws IOException {
```

Comment:
I think example activity also should change because of changes in sdk public interface.